.github/workflows/self-scheduled.yml

```
@@ -297,6 +297,57 @@ jobs:
           name: ${{ matrix.machine_type }}_run_tests_torch_cuda_extensions_gpu_test_reports
           path: /workspace/transformers/reports/${{ matrix.machine_type }}_tests_torch_cuda_extensions_gpu
 
+  run_tests_quantization_torch_gpu:
+    name: Quantization tests
+    strategy:
+      fail-fast: false
+      matrix:
+        machine_type: [single-gpu, multi-gpu]
+    runs-on: ['${{ matrix.machine_type }}', nvidia-gpu, t4, daily-ci]
+    container:
+      image: huggingface/transformers-quantization-latest-gpu
+      options: --gpus all --shm-size "16gb" --ipc host -v /mnt/cache/.cache/huggingface:/mnt/cache/
+    needs: setup
+    steps:
+      - name: Update clone
+        working-directory: /transformers
+        run: git fetch && git checkout ${{ github.sha }}
+
+      - name: Reinstall transformers in edit mode (remove the one installed during docker image build)
+        working-directory: /transformers
+        run: python3 -m pip uninstall -y transformers && python3 -m pip install -e .
```

```suggestion
      - uses: actions/checkout@v3
```
I think this can be completely removed in favor of uses: actions/checkout@v3: https://github.com/actions/checkout

See e.g.: https://github.com/huggingface/trl/blob/main/.github/workflows/slow-tests.yml#L33C15-L33C31

---

Thank you, I will first test it without and try to add that when I'm able to run the workflow successfully !

---

You also need to call `pip install -e .` at the begining of the job after the actions/checkout

---

I will leave it as it is since I think that @ydshieh prefer not to use `actions/checkout@v3`. We can still make the change in this [PR](https://github.com/huggingface/transformers/pull/29197) if we see that it works !

---

yes, I talked to @younesbelkada . Il fait n'importe de quoi 😆  (joking)

---

hahaha